### ISSUE TITLE
Unable to install selenium 3.0 beta

### ISSUE THREAD

**Issue by ghost** [vvo/selenium-standalone#218]
When trying to install the 3.0 selenium beta 3 I get the following error.

`Receive Error: Could not request headers from https://selenium-release.storage.googleapis.com/3.0/selenium-server-standalone-3.0.0-beta3.jar`

It seems the -beta3 is being removed from the selenium version. Working download url is https://selenium-release.storage.googleapis.com/3.0-beta3/selenium-server-standalone-3.0.0-beta3.jar

**Comment by vvo** [comment]
Yep that's right, do you think you could submit a PR?

**Comment by arun3528** [comment]
@vvo  I tried to 3.0.0 beta. @StepDav  try 3.0.0-beta3

 selenium.install({
      version: '3.0.0-beta3',
      baseURL: 'https://selenium-release.storage.googleapis.com',
      basePath: './lib/'
    },CB)

1) download was good on install
2)Tried to start and got an Error 

[Error: Selenium server did not start.Another Selenium process may already be running or your java version may be out of date.]

Any guesses what could have happened 

Note: When i try to run the jar file separately it works fine

**Comment by vvo** [comment]
> Any guesses what could have happened

No but you should be able to show the stderr of selenium by cloning this repo, npm link inside and try again by adding some console.log..

**Comment by StephenDavidson** [comment]
I am looking at whether or not the issue is upstream for me (Chimp). The package alone installs as expected from cli and nodejs.

**Comment by stephenash** [comment]
Running the start via cli instead of JS gives us a little bit more useful info.

```text
$ ./bin/selenium-standalone start --version=3.0.0-beta3
Exception in thread "main" com.beust.jcommander.ParameterException: Unknown option: -Dwebdriver.chrome.driver=/Users/sash/dev/git/selenium-standalone/.selenium/chromedriver/2.23-x64-chromedriver
       	at com.beust.jcommander.JCommander.parseValues(JCommander.java:742)
       	at com.beust.jcommander.JCommander.parse(JCommander.java:282)
       	at com.beust.jcommander.JCommander.parse(JCommander.java:265)
       	at com.beust.jcommander.JCommander.<init>(JCommander.java:210)
       	at org.openqa.grid.selenium.GridLauncherV3$1.setConfiguration(GridLauncherV3.java:210)
       	at org.openqa.grid.selenium.GridLauncherV3.buildLauncher(GridLauncherV3.java:138)
       	at org.openqa.grid.selenium.GridLauncherV3.main(GridLauncherV3.java:67)
/Users/sash/dev/git/selenium-standalone/bin/selenium-standalone:61
        throw err;
```

Looks like Selenium3 changed how arguments were parsed. Have a fix available in #224
